xep-0404.xml

```diff
@@ -65,8 +76,8 @@
   <section2 topic="Private Messages" anchor="concepts-pm">
     <p>
       &xep0369; exposes participant JIDs to other participants, and so messages can always be sent directly. When JIDs are hidden this is no longer possible.
-      Private messages MAY be sent to channel participants if allowed by channel policy. Private messages are
- addressed to the user's bare proxy JID determined from the participants node, and routed by the MIX to the user's bare real JID, where standard distribution rules will apply.
+      Private messages MAY be sent to channel participants if allowed by channel policy. Private messages are switched through the channel to hide addressing.
```

Comment:
Not just to hide addressing, but also to avoid being blocked as an unknown user, which is one of the reasons this doesn't belong in mix-anon.